third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -2700,13 +2700,41 @@ impl Frame {
                             )?;
                             eprintln!("trace abort({}): {}", error_code, str);
                         }
+                        let error =
+                            PartialVMError::new(StatusCode::ABORTED).with_sub_status(error_code);
```

Comment:
Previously, the message here was
```
format!("{} at offset {}", self.function.name_as_pretty_string(), self.pc)
```

I removed it in order to distinguish between aborts that include an explicit message and those that do not. This should be fine, since the message was discarded anyway when converting into an `ExecutionStatus`.

No information is lost: the `VMError` already records both the offset and abort location (see [abort.exp](https://github.com/aptos-labs/aptos-core/pull/18316/changes#diff-86fa0b16bfdc51af94f3fc5b706e1a8bdc2987f9e31e320cfc3f98f1e8450471)).

Reply:
Hmm that's interesting.

The only problem with what you suggested is I haven't introduced binary version 10 in this PR, that's the next PR. I'm starting to think maybe I shouldn't have split this into a separate PR.

I wonder if it would be better to introduce a new `ABORTED_WITH_MESSAGE` code. That way you can still have an `ABORTED` code with a message, but the message won't be propagated to the `AbortInfo`.